beets/dbcore/query.py

```diff
@@ -146,10 +146,9 @@ def __init__(self, field, fast=True):
     def col_clause(self):
         return self.field + " IS NULL", ()
 
-    @classmethod
     def match(self, item):
         try:
-            return item[self.field] is None
+            return item[self.field] is None or not item[self.field]
```

Comment:
If I got it right from reading
http://www.tutorialspoint.com/sqlite/sqlite_null_values.htm
Field IS NULL should return existing fields with nothing in them, so not those that do not have the field. Do I see that correct?
I that case the or "not item[field]" should disappear, correct?